packages/builder/src/registry.ts

```diff
@@ -274,7 +274,26 @@ export class OnChainRegistry extends CannonRegistry {
     }
   }
 
-  private async checkPackageOwnership(packageName: string) {
+  /**
```

Comment:
Adding a check for when a new package is being published and needs to be 'registered'. 
Subsequently register the package if the check doesnt pass.